src/main/java/ch/njol/skript/expressions/ExprWorldBorderWarningTime.java

```diff
@@ -0,0 +1,73 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.util.Timespan;
+import ch.njol.util.coll.CollectionUtils;
+import org.bukkit.WorldBorder;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Warning Time of World Border")
+@Description("The warning time of a world border. If the border is shrinking, the player's screen will be tinted red once the border will catch the player within this time period.")
+@Examples("set warning time of {_worldborder} to 1 second")
+@Since("INSERT VERSION")
+public class ExprWorldBorderWarningTime extends SimplePropertyExpression<WorldBorder, Timespan> {
+
+	static {
+		register(ExprWorldBorderWarningTime.class, Timespan.class, "[[world[ ]]border] warning time", "worldborders");
+	}
+
+	@Override
+	@Nullable
+	public Timespan convert(WorldBorder worldBorder) {
+		return new Timespan(worldBorder.getWarningTime() * 1000L);
```

Comment:
It might be easier to either explain what's going on here for why it's `* 1000L` or make use of the other timespan constructors of `new Timespan(TimePeriod timePeriod, Long time)`
ex: `new Timespan(TimePeriod.SECOND, worldBorder.getWarningTime())`